spec/lib/capistrano/configuration/plugin_installer_spec.rb

```diff
@@ -0,0 +1,103 @@
+require "spec_helper"
+require "capistrano/plugin"
+require "capistrano/scm/plugin"
+
+module Capistrano
+  class Configuration
+    class ExamplePlugin < Capistrano::Plugin
+      def set_defaults
+        set_if_empty :example_variable, "foo"
+      end
+
+      def define_tasks
+        task :example do
+          # intentionally blank
+        end
+
+        task :example_prerequisite do
+          # intentionally blank
+        end
```

Comment:
You could shorten these with `task(:example) {}`, but I'm not sure if that's any better.